libs/src/services/anchorAudiusData/index.ts

```diff
@@ -0,0 +1,918 @@
+import * as AudiusData from '@audius/anchor-audius-data'
+import type { AudiusDataProgram } from '@audius/anchor-audius-data'
+import anchor, { Idl } from '@project-serum/anchor'
+import type SolanaWeb3Manager from '../solanaWeb3Manager'
+import type Web3Manager from '../web3Manager'
+import { PublicKey, Keypair, SystemProgram, Transaction } from '@solana/web3.js'
+import SolanaUtils from '../solanaWeb3Manager/utils'
+
+type AnchorAudiusDataConfig = {
+  programId: string
+  adminPublicKey: string
+  adminStoragePublicKey: string
+}
+
+type OmitAndRequire<T, K extends keyof T, L extends keyof T> = Partial<
+  Omit<T, K>
+> &
+  Required<Pick<T, L>>
+
+/**
+ * AnchorAudiusData acts as the interface to solana auidus data programs from a client.
+ * It wraps methods to create transactions.
+ */
+export class AnchorAudiusData {
+  anchorAudiusDataConfig: AnchorAudiusDataConfig
+  solanaWeb3Manager: SolanaWeb3Manager
+  program: AudiusDataProgram | undefined
+  programId: PublicKey | undefined
+  adminPublicKey: PublicKey | undefined
+  adminStoragePublicKey: PublicKey | undefined
+  provider: anchor.Provider | undefined
+  web3Manager: Web3Manager
+
+  /**
+   * @param {Object} anchorAudiusDataConfig
+   *  the solana cluster RPC endpoint
```

Comment:
is the above referring to a param that isn't documented here?

Reply:
Yes, I need to update docs as well